src/imputation/MoR.py

```diff
@@ -140,7 +152,7 @@ def carry_forwards(df, backdata, impute_vars):
 
     df.loc[match_cond, "imp_marker"] = "CF"
 
-    df.loc[match_cond] = create_imp_class_col(df, "200_prev", "201_prev")
+    # df.loc[match_cond] = create_imp_class_col(df, "200_prev", "201_prev")
```

Comment:
Am I correctly thinking that in the previous version you were creating an imputation class, and now you are repeating it from the last year? Are there any references that switch from one imp class to another, and what's the methodology advice on that?